doc/sphinx/language/core/conversion.rst

```diff
@@ -99,12 +93,12 @@ destroyed, this reduction differs from δ-reduction. It is called
    E[Γ] ⊢ \letin{x}{u:U}{t}~\triangleright_ζ~\subst{t}{x}{u}
 
 
-.. _eta-expansion:
+.. _eta-expansion-sect:
 
 η-expansion
 ~~~~~~~~~~~
 
-Another important concept is η-expansion. It is legal to identify any
+Another important concept is :gdef:`eta-expansion`. It is legal to identify any
```

Comment:
I'd suggest a uniform style
```suggestion
Another important concept is :gdef:`η-expansion <eta-expansion>`. It is legal to identify any
```

Reply:
I guess.  Greek letters in glossary terms are awkward: we want to list them in the index in Romanized form, hence the "<...>", but then if you want to reference the term with the greek letter you have to give the verbose ```:term:`η-expansion <eta-expansion>` ```